functions/mqhandlers.go

```diff
@@ -494,9 +494,9 @@ func getServerBrokerStatus() (bool, error) {
 		return false, err
 	}
 
-	if _, ok := response["is_broker_conn_open"]; ok {
-		return response["is_broker_conn_open"].(bool), nil
-	}
+	// if _, ok := response["is_broker_conn_open"]; ok {
+	// 	return response["is_broker_conn_open"].(bool), nil
+	// }
```

Comment:
why did we comment this?
On client side, it just gives accurate conn status with is_broker_conn_open, then it can pull over api

Reply:
During the test, it caused a lot of mqfallback.  Actually,  the connection is not lost, it's in reconnecting status.

The server status api is called every 30 seconds.  When there are  1000 hosts in the network,   the server status api is called anytime, never have a rest.  When there is a moment the mq connection is in reconnecting status,  it will give client a false alarm that the mq is not available. Then a mqfallback is called in client side. It results in unnecessary mqfallback.